packages/expect/src/asymmetric_matchers.js

```diff
@@ -146,34 +153,50 @@ class ArrayContaining extends AsymmetricMatcher {
 class ObjectContaining extends AsymmetricMatcher {
   sample: Object;
 
-  constructor(sample: Object) {
+  constructor(sample: Object, inverse: boolean = false) {
     super();
     this.sample = sample;
+    this.inverse = inverse;
   }
 
   asymmetricMatch(other: Object) {
     if (typeof this.sample !== 'object') {
       throw new Error(
-        "You must provide an object to ObjectContaining, not '" +
+        `You must provide an object to ${this.toString()}, not '` +
```

Comment:
do we want the `toString`?

Reply:
Yes, so the error message is more intelligible. I think at least.